src/main/java/com/zaxxer/nuprocess/internal/BasePosixProcess.java

```diff
@@ -317,6 +369,62 @@ else if (WIFSIGNALED(status)) {
       return this;
    }
 
+   private int spawnWithCwdSetViaPthreadChdir(
```

Comment:
The code is self-explanatory, so the method does not need to be so descriptive.  ``spawnOsxWithCwd()``.